runtime/src/storage/mkvs/tree/lookup.rs

```diff
@@ -78,6 +95,10 @@ impl Tree {
             }
             NodeKind::Internal => {
                 let node_ref = node_ref.unwrap();
+                if let Some(pb) = proof_builder.as_mut() {
+                    pb.include(&node_ref.borrow());
+                }
```

Comment:
Shouldn't this happen for all nodes not only internal nodes?

Reply:
I believe that, since we don't have the "include siblings" option (and we also only support "existence proofs" and not non-existence) , it's identical if this is done here, or above this match?
Because:
- if we reach a Nil node then we won't return a proof anyway
- Internal nodes are already "included" with leaf nodes

Maybe i'm missing something.